### PR TITLE
Chapter 12: Password reset

### PR DESCRIPTION
**Pull request by khoapn-1940**
image

### CODE REVIEW

**Comment by khoapn-1940** [comment]
ready

File: app/assets/javascripts/password_resets.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (phamvanbk11) [inline app/assets/javascripts/password_resets.coffee]: Xóa file

File: app/assets/stylesheets/password_resets.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the PasswordResets controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (phamvanbk11) [inline app/assets/stylesheets/password_resets.scss]: Xóa file

File: app/views/password_resets/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(:password_reset, url: password_resets_path) do |f| %>
+      <%= f.label :email %>
+      <%= f.email_field :email, class: 'form-control' %>
+
+      <%= f.submit "Submit", class: "btn btn-primary" %>
+    <% end %>
+  </div>
+</div>
```
Comment (phamvanbk11) [inline app/views/password_resets/new.html.erb]: Thêm dòng trống cuối file

File: config/routes.rb [hunk 4]
```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'password_resets/new'
+  get 'password_resets/edit'
```
Comment (phamvanbk11) [inline config/routes.rb]: Nháy "

**Comment by phamvanbk11** [comment]
commented

**Comment by khoapn-1940** [comment]
ready

**Comment by phamvanbk11** [comment]
merged

**Pull request merged**